src/public/css/styles.css

```diff
@@ -0,0 +1,95 @@
+/* app colors */
+/* font sizes */
+.theme, html,
+body {
+  background-color: #323232;
```

Comment:
Is it confusing to copy the look and feel of CHT? Or is that what we want to do here? 

I'm concerned that user managers sometimes have to use the cht and sometimes they have to use this management tool; and if they look too similar this might be confusing for some of them.  

Maybe we shouldn't match one prominent thing like the background color?  I also think this color is difficult since we have text in the logos.  You're using a white font on this background; but the logo for Kenya has black font.  We can't use a white font in the logo because it is also used on the other pages which have a white background.